deps/simdutf/simdutf.h

```diff
@@ -572,7 +574,7 @@ SIMDUTF_DISABLE_UNDESIRED_WARNINGS
 #define SIMDUTF_SIMDUTF_VERSION_H
 
 /** The version of simdutf being used (major.minor.revision) */
-#define SIMDUTF_VERSION "3.2.3"
+#define SIMDUTF_VERSION "3.2.4"
```

Comment:
This doesn't seem to match the PR/commit title (3.2.7)?

Reply:
@lemire This seems to be wrong

Reply:
Welp, it is how it is in the official repo 🤷

Reply:
Indeed. It should be fixed now.